taichi/program/conjugate_gradient.cpp

```diff
@@ -0,0 +1,94 @@
+#include "conjugate_gradient.h"
+
+namespace taichi::lang {
+void CUCG::init_solver() {
+#if defined(TI_WITH_CUDA)
+  if (!CUBLASDriver::get_instance().is_loaded()) {
+    bool load_success = CUBLASDriver::get_instance().load_cublas();
+    if (!load_success) {
+      TI_ERROR("Failed to load cublas library!");
+    }
+  }
+  CUBLASDriver::get_instance().cubCreate(&handle);
+  // int version;
+  // CUBLASDriver::get_instance().cubGetVersion(handle, &version);
+  // fmt::print("CUBLAS version: {}\n", version);
```

Comment:
Are we intended to keep these comments?

Reply:
I'm not sure if we should print out the version of CuBlas.

Reply:
We can print info like this using `TI_TRACE`.